checks/ping.go

```diff
@@ -72,6 +72,7 @@ func (r *PingResult) Metrics(t time.Time, check *m.CheckWithSlug) []*schema.Metr
 			TargetType: "gauge",
 			Time:       t.Unix(),
 			Tags: []string{
+				"stat:min",
```

Comment:
this is redundant information.  The "Metric" field already describes this measurement as being the "min"